iocore/net/ProxyProtocol.cc

```diff
@@ -55,7 +56,11 @@ constexpr uint8_t PPv2_PROTO_UNIX_DATAGRAM = 0x32;
 
 constexpr uint16_t PPv2_ADDR_LEN_INET  = 4 + 4 + 2 + 2;
 constexpr uint16_t PPv2_ADDR_LEN_INET6 = 16 + 16 + 2 + 2;
-// constexpr uint16_t PPv2_ADDR_LEN_UNIX  = 108 + 108;
+constexpr uint16_t PPv2_ADDR_LEN_UNIX  = 108 + 108;
+
+constexpr size_t TS_IPv4_LEN = sizeof(in_addr_t); ///< 4
```

Comment:
This should go in "ink_inet.h" next to `TS_IP6_SIZE`.

Reply:
Moved